app/assets/stylesheets/components/_footer_nav.scss

```diff
@@ -0,0 +1,23 @@
+.footer {
```

Comment:
.footer is a bootstrap class already - maybe call this something more specific like abc-footer?

Reply:
done